ui-components/src/helpers/filters.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  ListingFilterParams,
+  ListingFilterKeys,
+  EnumListingFilterParamsComparison,
+} from "@bloom-housing/backend-core/types"
+import { ParsedUrlQuery } from "querystring"
+
+function getComparisonForFilter(filterKey: ListingFilterKeys) {
+  switch (filterKey) {
+    case ListingFilterKeys.name:
+      return EnumListingFilterParamsComparison["="]
+    case ListingFilterKeys.neighborhood:
+      return EnumListingFilterParamsComparison["="]
+    case ListingFilterKeys.status:
+      return EnumListingFilterParamsComparison["="]
```

Comment:
style nit: fwiw I like to let things that have the same return value fall through, instead of having a return for each one, but it's totally up to you.

so like
```suggestion
    case ListingFilterKeys.name:
    case ListingFilterKeys.neighborhood:
    case ListingFilterKeys.status:
      return EnumListingFilterParamsComparison["="]
```